Cargo.toml

```diff
@@ -52,6 +52,7 @@ stm32l4x2 = ["stm32l4/stm32l4x2"]
 stm32l4x3 = ["stm32l4/stm32l4x3"]
 stm32l4x5 = ["stm32l4/stm32l4x5"]
 stm32l4x6 = ["stm32l4/stm32l4x6"]
+stm32l47x = ["stm32l4x6"]
```

Comment:
At the moment there aren't any plans to support boards by themselves

Reply:
I should add, mainly because it's quite time consuming to get everything behind a feature gate. If you could get some more stuff behind feature gates for your board (usarts, gpio etc), I would definitely be interested in having your board as the first one! :)

Reply:
Ok the idea here is not board specific. The stm32l476 and stm32l475 does not have the HSI48 clock. Looking a the datasheet the HSI48 clock seems to be mainly for the USB, RNG, ... clock source. So Mi idea was to abstract this mater of fact so that we chose the best clock source depending on the MCU capability.